x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -89,38 +89,55 @@ datasources:
          - id?: {id}
            enabled?: false # default true
            metricset: cpu
-           period: 10s
            dataset: system.cpu
            metrics: ["percentages", "normalized_percentages"]
+           period: 10s
          - metricset: memory
            dataset: system.memory
+           period: 10s
          - metricset: diskio
            dataset: system.diskio
+           period: 10s
          - metricset: load
            dataset: system.load
+           period: 10s
          - metricset: memory
            dataset: system.memory
+           period: 10s
          - metricset: process
            dataset: system.process
            processes: ["firefox*"]
+           include_top_n:
```

Comment:
is this something we understand or is it input/processor config?

Reply:
This is something we relay to the input section in metricbeat. This required a bit of massaging:

https://github.com/elastic/beats/blob/69cd8d89dfebec5410746e086fe0c772f58d2662/metricbeat/modules.d/system.yml#L19-L22

Reply:
in this case it's fine